src/Winton.Extensions.Configuration.Consul/ConsulConfigurationProvider.cs

```diff
@@ -11,161 +11,161 @@
 
 namespace Winton.Extensions.Configuration.Consul
 {
-    /// <summary>
-    ///     Each instance loads configuration for the key in Consul that is specified in
-    ///     the contained <see cref="IConsulConfigurationSource" />.
-    ///     It has the ability to automatically reload the config if it changes in Consul.
-    /// </summary>
-    /// <remarks>
-    ///     Each instance maintains its own <c>lastIndex</c> and uses this to detect changes.
-    ///     Each instance ensures calls to Consul are serialised, to avoid concurrent access to <c>lastIndex</c>.
-    /// </remarks>
-    internal sealed class ConsulConfigurationProvider : ConfigurationProvider, IDisposable
+  /// <summary>
```

Comment:
I think your IDE might have auto-formatted this file to make the indentation 2 spaces, rather than 4. Could you revert?

Reply:
rebased and fixed each commit